util.go

```diff
@@ -65,3 +66,19 @@ func Log(level LogLevel, msg string, args ...interface{}) {
 		os.Exit(1)
 	}
 }
+
+var statsd_client statsd.Statter
+
+func Gauge(bucket string, value int64, rate float32) error {
+	if statsd_client != nil {
+		return statsd_client.Gauge(bucket, value, rate)
+	}
+	return nil
+}
+
+func Inc(bucket string, value int64, rate float32) error {
+	if statsd_client != nil {
+		return statsd_client.Inc(bucket, value, rate)
+	}
+	return nil
+}
```

Comment:
Are the `statsd_client.Gauge/Inc` methods documented as thread-safe? If not, we should guard these functions with a mutex.